src/main/java/org/cryptomator/ui/error/ErrorController.java

```diff
@@ -46,16 +62,28 @@ public class ErrorController implements FxController {
 	private final Stage window;
 	private final Environment environment;
 
-	private BooleanProperty copiedDetails = new SimpleBooleanProperty();
+	private final BooleanProperty copiedDetails = new SimpleBooleanProperty();
+	private final ObjectProperty matchingErrorDiscussion = new SimpleObjectProperty();
```

Comment:
```suggestion
	private final ObjectProperty<ErrorDiscussion> matchingErrorDiscussion = new SimpleObjectProperty<>();
```

(so we don't need type casts further down)

Reply:
nice, it's implemented